CONTRIBUTING.md

```diff
@@ -82,15 +82,15 @@ With this remote setting, you can check your git remote configuration like this:
 $ git remote -v
 origin     git@github.com:<your-username>/seata-go.git (fetch)
 origin     git@github.com:<your-username>/seata-go.git (push)
-upstream   git@github.com:seata/seata-go.git (fetch)
+upstream   git@github.com:apache/seata-go.git (fetch)
 upstream   no-pushing (push)
 ```
 
 Adding this, we can easily synchronize local branches with upstream branches.
 
 ### Branch Definition
 
-Right now we assume every contribution via pull request is for [branch develop](https://github.com/seata/seata-go/tree/develop) in seata-go. Before contributing, be aware of branch definition would help a lot.
+Right now we assume every contribution via pull request is for [branch develop](https://github.com/apache/seata-go/tree/develop) in seata-go. Before contributing, be aware of branch definition would help a lot.
```

Comment:
For the replacement related to the documentation, I believe it is necessary to use apache/incubator-seata-go to ensure the URL is accessible. It is recommended to replace it module by module.

Reply:
There is a problem with the documentation here, it should be the dev branch, not develop.
At the same time, the dev branch and the master branch are lagging far behind, and I think they should be maintained.

Reply:
https://github.com/apache/seata-go will redirect to https://github.com/apache/incubator-seata-go, it is setted in github

Reply:
> https://github.com/apache/seata-go will redirect to https://github.com/apache/incubator-seata-go, it is setted in github

https://github.com/apache/seata-go will redirect to https://github.com/apache/incubator-seata-go, it is setted in github

Reply:
> There is a problem with the documentation here, it should be the dev branch, not develop. At the same time, the dev branch and the master branch are lagging far behind, and I think they should be maintained.
i have changed it to master